src/main/java/seedu/address/model/person/Interaction.java

```diff
@@ -22,13 +22,12 @@ public static enum Outcome {
         * Returns true if a given string is a valid outcome.
         */
         public static boolean isValidOutcome(String test) {
-            System.out.println(test);
             return test.matches("INTERESTED|NOT_INTERESTED|FOLLOWUP_REQUIRED|UNKNOWN");
         }
     }
 
-    public final String interactionNote;
-    public final Outcome outcome;
+    private final String interactionNote;
```

Comment:
I think we need getters for these now right?

Reply:
Nope, iinw its just the toString that's really being used

Reply:
Edit: Yeap, fixed in PR #52